### ISSUE TITLE
Bug: Temporary view shown before node has started

### ISSUE THREAD

**Issue by danielnordh** [reez/Monday#144]
When a user is onboarded, StartView briefly shows a screen with a button offering a way to go back to Onboarding.
It happens both on first run, and subsequent app starts.
This could be a regression from my work in #131 

Here is the UI Code being shown:

<img width="811" alt="image" src="https://github.com/user-attachments/assets/8241c39c-7ea2-48fa-8b6f-9ec274afcc9d" />

It seems to be caused by BitcoinView not being launched until node is started, and the start is not triggered until StartView .onAppear.

I'll look into how to avoid this happening.
Potentially having LDKMondayApp observing whether a node is started instead of whether 'onboarding' is true.
This could enable bypassing StartView completely and showing BitcoinView when node is started.

If that is not possible, or a good idea we can do a visual workaround in StartView.

**Comment by reez** [comment]
I don't think this is a bug just a weird workaround needed previously. I'll add some more context in a bit

**Comment by reez** [comment]
Context: same time period that I added the "reset preferences" stuff we discussed yesterday, similar issue(s) to try to work around

Problem: Esplora not working

(Temporary) Solution: Give a "loading" screen that is minimally noticeable when things work right, but if things aren't working if node is stuck loading (because the Esplora provided to it isn't working, and client app may not know that) user can get back to the start of onboarding to select a different Esplora.

Reproduce: to reproduce what would happen in this case you can mimic it by changing `https://mutinynet.com/api` to `https://mutinyne.com/api` (remove "t")

Further thoughts: definitely not the ideal solution, but at the time did not know if the underlying Esplora problem would be a a temporary problem or long term problem. I think we touched on some overall onboarding thoughts yesterday with breaking out the `build` from `init` https://github.com/reez/Monday/blob/main/LDKNodeMonday/Service/Lightning%20Service/LightningNodeService.swift#L88, which would further the ability for what would probably be a better onboarding where some of these steps could be broken out a bit more (example: if esplora is the problem, break out esplora selection into a separate screen where the underlying functionality can handle an esplora problem before passing a non-working esplora into the node builder). What was extra tricky about this at the time was the app would not crash, but would also not get notified of an error with the esplora either.

![Simulator Screenshot - iPhone 16 Pro - 2024-12-20 at 08 24 00](https://github.com/user-attachments/assets/4a5f8f1b-8f2f-4a54-84df-33ea0b9bf4fa)

**Comment by danielnordh** [comment]
If we go forward with #145 this view is replaced with a loading view.

**Comment by danielnordh** [comment]
Closed by #150
